src/lcs.nim

```diff
@@ -1,20 +1,19 @@
-import os, parseopt, tables, os, strutils, utils
+import std/[parseopt, tables, strutils]
```

Comment:
`std` prefix is generally preferred to be used when importing stdlib modules. Some stdlib modules can **only** be imported with it